src/config.c

```diff
@@ -61,6 +61,23 @@ CONFIG_GETTER(getMinPrefix) {
   return sdscatprintf(ss, "%lld", config->iteratorsConfigParams.minTermPrefix);
 }
 
+// MINSTEMLEN
+CONFIG_SETTER(setMinStemLen) {
+  long long minStemLen;
+  int acrc = AC_GetLongLong(ac, &minStemLen, AC_F_GE1);
+  if (minStemLen < MIN_MIN_STEM_LENGHT) {
+    QueryError_SetErrorFmt(status, MIN_MIN_STEM_LENGHT, "Minimum stem length cannot be lower than %lld", MIN_MIN_STEM_LENGHT);
```

Comment:
Consider using `AC_ERR_ELIMIT`, which is used for most config-param bounds errors.
This is more explicit which is nice too, but less coherent with the existing similar errors.
Your call.

Reply:
OK. Change done. Thanks.